Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
     <string name="files_base_dir_does_not_exist"><![CDATA[Base directory for files does not exist! Set it manually in the settings (Media -> Save location)]]></string>
-    <string name="image_saver_could_not_figure_out_save_location">Couldn\'t figure out save location</string>
-    <string name="settings_captcha_setup">Captcha Settings</string>
-    <string name="settings_captcha_setup_description">Captcha settings are per-site configurable. Tap for site setup.</string>
+    <string name="image_saver_could_not_figure_out_save_location">Could not figure out where to save this</string>
+    <string name="settings_captcha_setup">CAPTCHA Settings</string>
+    <string name="settings_captcha_setup_description">Tap to set up CAPTCHA settings for each site.</string>
     <string name="image_saver_canceled_by_user">Canceled by user</string>
     <string name="image_saver_unknown_location">Unknown location</string>
-    <string name="image_saver_failed_to_save_image">Failed to save the image. Reason: %s</string>
-    <string name="image_saver_no_write_permission">Cannot start saving images without the WRITE_EXTERNAL_STORAGE permission</string>
+    <string name="image_saver_failed_to_save_image">Could not save the image because %s</string>
+    <string name="image_saver_no_write_permission">Grant write permission to external storage to save images</string>
     <string name="media_settings_image_click_preload_strategy_name">Image preload strategy</string>
     <string name="media_settings_image_click_preload_strategy_description">When opening up an image pager, determines what image(s) should be preloaded next.</string>
-    <string name="setting_image_viewer_gestures">Image Viewer gestures</string>
-    <string name="setting_image_viewer_gestures_description">Adds ability to use gestures to close (swipe-up) and save (swipe-bottom) an image/gif/webm when viewing them in gallery</string>
+    <string name="setting_image_viewer_gestures">Image viewer gestures</string>
+    <string name="setting_image_viewer_gestures_description">Allows using gestures to close (swipe-↑) and save (swipe-↓) an image, GIF, or WebM in gallery view.</string>
 
-    <string name="lorem_ipsum">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</string>
-    <string name="report_controller_report_an_error_problem">Report an error/crash/other problem</string>
+    <string name="lorem_ipsum">Fearful ubermensch war insofar love horror. Grandeur convictions prejudice snare pinnacle against revaluation spirit. Love love decrepit.</string>
```

Comment:
Why?

Reply:
It is a http://nietzsche-ipsum.com/ :) The lorem ipsum is the choice of those that dare not think.